packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -181,11 +181,13 @@ def is_correct_contract(
             is not ContractApiMessage.Performative.STATE
         ):
             log_message = self.LogMessages.failed_verification
-            self.context.logger.info(f"{log_message}")
-            return False
-        log_message = self.LogMessages.response_verification
+            verified = False
+        else:
+            log_message = self.LogMessages.response_verification
+            verified = cast(bool, contract_api_response.state.body["verified"])
+
         self.context.logger.info(f"{log_message}: {contract_api_response}")
-        return cast(bool, contract_api_response.state.body["verified"])
+        return verified
```

Comment:
Not related to this PR, but it is useful to know the response in case of unexpected performative.